### PR TITLE
exonerate: init at 2.4.0

### PR DESCRIPTION
**Pull request by bzizou**
###### Motivation for this change
Tool for biologists using our computing center

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/science/biology/exonerate/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name = "exonerate-${version}";
+
+  src = fetchurl {
+    url = "http://ftp.ebi.ac.uk/pub/software/vertebrategenomics/exonerate/${name}.tar.gz";
+    sha256 = "0hj0m9xygiqsdxvbg79wq579kbrx1mdrabi2bzqz2zn9qwfjcjgq";
+  };
+
+  doCheck = true;
+
+  buildInputs = [ glib pkgconfig ];
```
Comment (jtojnar) [inline pkgs/applications/science/biology/exonerate/default.nix]: Programs executed during build should go to `nativeBuildInputs`.

Comment (bzizou) [reply pkgs/applications/science/biology/exonerate/default.nix]: Yes, I fixed that, thanks!

File: pkgs/applications/science/biology/exonerate/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, glib, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "2.4.0";
+  name = "exonerate-${version}";
```
Comment (aanderse) [inline pkgs/applications/science/biology/exonerate/default.nix]: `name` -> `pname` please.

**Review by aanderse — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/biology/exonerate/default.nix (hunk 2 above)
Comment (bzizou) [reply pkgs/applications/science/biology/exonerate/default.nix]: Done!

**Review by aanderse — APPROVED** [review]
:+1: LGTM

**Comment by aanderse** [comment]
@GrahamcOfBorg build exonerate

**Comment by aanderse** [comment]
Thanks for your contribution!

**Pull request merged**